### PR TITLE
Feature/improve annotations loading flow

### PR DESCRIPTION
**Pull request by niko-8d4**
When a user navigates to another chapter, both navigate and handleIFrameLoad are being called. There is some overlap between the two functions and e.g. the annotations are being drawn multiple times. Some of those happens too early.

This PR makes it so that navigating between chapters only redraws the annotations once and allows to specifically skip redrawing of annotations when the book view is updated (only relevant for handleIFrameLoad for now).

### CODE REVIEW

File: src/modules/AnnotationModule.ts [hunk 1]
```diff
@@ -492,7 +492,9 @@ export class AnnotationModule implements ReaderModule {
   public showHighlights() {
```
Comment (aferditamuriqi) [inline src/modules/AnnotationModule.ts]: @nikolai-ri i missed this when we spoke, we have to revert the change on the highlights list in this method, as this method is solely responsible for showing all highlights in the toc area , see dita reader example.

Comment (niko-8d4) [reply src/modules/AnnotationModule.ts]: Thanks & good catch!
I reverted the change in showHighlights to get the annotations of all chapters and can confirm its working in the reader example project.

**Review by aferditamuriqi — APPROVED** [review]
